homeassistant/components/husqvarna_automower/lawn_mower.py

```diff
@@ -72,15 +82,54 @@ async def async_setup_entry(
     platform.async_register_entity_service(
         "override_schedule_work_area",
         {
-            vol.Required("work_area_id"): vol.Coerce(int),
-            vol.Required("duration"): vol.All(
+            vol.Required(CONF_WORK_AREA_ID): vol.Coerce(int),
+            vol.Required(CONF_DURATION): vol.All(
                 cv.time_period,
                 cv.positive_timedelta,
                 vol.Range(min=timedelta(minutes=1), max=timedelta(days=42)),
             ),
         },
         "async_override_schedule_work_area",
     )
+    platform.async_register_entity_service(
+        "set_schedule",
+        {
+            vol.Required("mode"): vol.All(
+                cv.string,
+            ),
```

Comment:
what modes?

Reply:
done in https://github.com/home-assistant/core/pull/126311/commits/8b5ed94c688b46d84f69501229f99622b48173b7